--- ./src/metamail/uue.c.orig	2010-08-11 20:04:47.000000000 -0500
+++ ./src/metamail/uue.c	2010-08-11 20:05:12.000000000 -0500
@@ -30,7 +30,7 @@
 }
 
 
-getline (buf, size, fp)
+uue_getline (buf, size, fp)
 char *buf;
 int size;
 FILE *fp;
@@ -70,7 +70,7 @@
     char buf[63];
 
     while (1) {
-	if (getline (buf, sizeof buf, infp) < 0) {
+	if (uue_getline (buf, sizeof buf, infp) < 0) {
 	    fprintf (stderr, "Premature EOF!\n");
 	    return;
 	}
@@ -82,7 +82,7 @@
 	}
     }	
     while (1) {
-	if (getline (buf, sizeof buf, infp) < 0) {
+	if (uue_getline (buf, sizeof buf, infp) < 0) {
 	    fprintf (stderr, "Premature EOF!\n");
 	    return;
 	}
