From 7b7ad9f3e8dc91d08050c7b24ecae603b64ca52f Mon Sep 17 00:00:00 2001
From: Nicola Fontana <ntd@entidi.it>
Date: Sat, 18 Feb 2012 15:14:22 +0100
Subject: [PATCH 2/2] Extend commit #c2232d50 to "Send to..."

---
 ROX-Filer/src/menu.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ROX-Filer/src/menu.c b/ROX-Filer/src/menu.c
index bf4bae3..a9ec107 100644
--- a/ROX-Filer/src/menu.c
+++ b/ROX-Filer/src/menu.c
@@ -1674,7 +1674,7 @@ static void show_send_to_menu(GList *paths, GdkEvent *event)
 
 	send_to_paths = paths;
 
-	g_signal_connect(menu, "unmap_event", G_CALLBACK(menu_closed), NULL);
+	g_signal_connect(menu, "selection-done", G_CALLBACK(menu_closed), NULL);
 
 	popup_menu = menu;
 	show_popup_menu(menu, event, 0);
-- 
1.7.9.1

